### DISCUSSION TITLE
many-to-many matching not allowed

### DISCUSSION THREAD

**Discussion by seaurching** [grafana/mimir#3390] [Help and support]
#### Describe the bug
group_left data  tips error:
```
irate(container_network_receive_bytes_total{namespace="kube-system"}[4h]) *  on (pod,namespace) group_left(workload,workload_type) namespace_workload_pod:kube_pod_owner:relabel{namespace="kube-system"}
```
![image](https://user-images.githubusercontent.com/7349963/199916808-bf45f7c7-5bb3-4198-afe3-f53a5a03d47b.png)
#### Environment
- mimir version
  -  v2.2.0

**Comment by pracucci** [comment]
"Many to many" vector matching is not supported in PromQL. You can learn more about vector matching here:
https://prometheus.io/docs/prometheus/latest/querying/operators/#vector-matching

**Reply by seaurching** [reply]
this promql is normal on official 2.39.1，could show result correctly.
上面的那个语句在官方2.39.1版本可以查询到数据
